cypress/support/commands.ts

```diff
@@ -0,0 +1,37 @@
+/// <reference types="cypress" />
```

Comment:
This is great and helpful, love seeing that you're thinking about using cypress commands!